steps/triggers/triggers.go

```diff
@@ -2,15 +2,33 @@ package triggers
 
 import (
 	"github.com/getgauge-contrib/gauge-go/gauge"
+	m "github.com/getgauge-contrib/gauge-go/models"
 	"github.com/openshift-pipelines/release-tests/pkg/store"
 	"github.com/openshift-pipelines/release-tests/pkg/triggers"
 )
 
 var _ = gauge.Step("Expose Event listener <elname>", func(elname string) {
 	routeurl := triggers.ExposeEventListner(store.Clients(), elname, store.Namespace())
 	store.PutScenarioData("route", routeurl)
+	store.PutScenarioData("elname", elname)
 })
 
 var _ = gauge.Step("Mock get event", func() {
-	triggers.MockGetEvent(store.GetScenarioData("route"))
+	gauge.GetScenarioStore()["response"] = triggers.MockGetEvent(store.GetScenarioData("route"))
+})
+
+var _ = gauge.Step("Assert eventlistener response", func() {
+	triggers.AssertElResponse(store.Clients(), store.HttpResponse(), store.GetScenarioData("elname"), store.Namespace())
+})
+
+var _ = gauge.Step("Cleanup Triggers", func() {
+	triggers.CleanupTriggers(store.Clients(), store.GetScenarioData("elname"), store.Namespace())
+})
+
+var _ = gauge.Step("Mock post event <eventpayloads>", func(eventpayloads *m.Table) {
```

Comment:
Can we implement this step in a regular way (without using table)? I feel adding table reduces the readability of the step.
Though we have added table, in the test scenario steps we have used a table with only one row

Reply:
Reason behind this table format is, later in future you can send multiple requests in parallel, and any ideas about it?

Instead of having multiple arguments it's safe to have a table ? WDYT?

Reply:
> Reason behind this table format is, later in future you can send multiple requests in parallel, and any ideas about it?

We will still be able to do that with repeating steps with different parameters or we can use [this](https://docs.gauge.org/writing-specifications.html?os=linux&language=python&ide=vscode#table-driven-scenario) feature ( If it's supported :D)